### DISCUSSION TITLE
Taro build --type weapp失败，git bash提示Segmentation fault

### DISCUSSION THREAD

**Discussion by zhangjing1992** [NervJS/taro#13598] [Q & A] [answered]
如题，运行`taro build --type weapp`失败，命令行只打印了版本信息，没有任何错误输出，dist文件夹只有一个`project.config.json`文件
![image](https://user-images.githubusercontent.com/43285899/230536171-13d31667-e84f-4374-861c-062447f3baf1.png)
运行taro doctor显示没有问题
![image](https://user-images.githubusercontent.com/43285899/230536300-41de50fd-f4f5-4bd7-a9cb-745192ea88c3.png)
使用`Git Bash`运行`taro build --type weapp`提示`Segmentation fault`

**Comment by TheKonka** [comment]
方便的话可以给个可以复现的仓库

**Reply by zhangjing1992** [reply]
应该是环境的问题，我换台电脑就可以了

**Reply by zhangjing1992** [reply]
使用git bash运行taro build --type weapp提示Segmentation fault

**Reply by zhangjing1992** [reply, marked as answer]
解决了，是因为clone下来的库没有taro init生成的.swc文件夹，复制一个过来就好了

**Comment by zhangjing1992** [comment]
解决了，是因为clone下来的库没有taro init生成的.swc文件夹，复制一个过来就好了

**Reply by TheKonka** [reply]
奇怪🤔️，clone 下来的库没有.swc 是正常的，你删除后重新安装运行应该会自动生成
